src/components/routeComponents/memberInformationGrid/attendanceResportModal/AttendanceReportModal.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Modal from 'pepcus-core/lib/Modal';
+import Box from 'pepcus-core/lib/Box';
+import ModalHeader from 'components/routeComponents/memberInformationGrid/attendanceResportModal/ModalHeader';
+import Accordion from 'components/common/Accordion/Accordion';
+import AttendanceDisplayCard from 'components/routeComponents/memberInformationGrid/attendanceResportModal/AttendanceDisplayCard';
+import PropTypes from 'prop-types';
+import { getConstants } from 'reducers/constants';
+import { connect } from 'react-redux';
+
+const AttendanceReportModal = ({ constants }) => {
+	const { ATTENDANCE_REPORT } = constants;
+	const [isModalOpen, setIsModalOpen] = useState(false);
+
+	const openModal = () => {
+		setIsModalOpen(true);
+	};
+
+	const closeModal = () => {
+		setIsModalOpen(false);
+	};
+
+	const renderModal = () => {
+		if (isModalOpen) {
+			return (
+				<Modal open={isModalOpen} style={{ padding: '0' }}>
+					<StyledBox
+						backgroundColor="modal"
+						padding="0"
+						borderColor="tertiary"
+						margin="0"
+						width="auto">
+						<ModalHeader onclick={closeModal} />
+						<StyledModalBody>
+							<Accordion
+								title={
+									<div>
+										Day1 <span>combination attendance</span>
```

Comment:
We should get these strings from JSON constants
```suggestion
										Day1 <span>combined attendance</span>
```

Reply:
Resolved